core/src/com/unciv/ui/utils/CameraStageBaseScreen.kt

```diff
@@ -71,19 +73,22 @@ open class CameraStageBaseScreen : Screen {
     }
 
     companion object {
-        var skin = Skin(Gdx.files.internal("skin/flat-earth-ui.json"))
+        var skin = Skin().apply {
+            add("Nativefont", Fonts().getFont(45, true), BitmapFont::class.java)
```

Comment:
you can see this code.

Reply:
Yes, but what does this actually change? In what way does it improve the game to have this as part of the skin?

Reply:
if we use this flat-earth-ui.png don't need to contain font picture.
you can see this:
 https://stackoverflow.com/questions/24856201/putting-freetypefont-into-libgdx-skin
i also edit bitmapfont in file "android/assets/skin/flat-earth-ui.json".

Reply:
You're missing the point...
I get why to remove the existing font from the skin, I don't get why we need to add our font to the skin. We can just not add the NativeFont to the skin at all.

Reply:
skin must include bitmapfont,  you can look at 
 "android/assets/skin/flat-earth-ui.json", 
for example, com.badlogic.gdx.scenes.scene2d.ui.TextButton$TextButtonStyle has a key "font", if we don't add the NativeFont to the skin, we should not delete "*.fnt". if not, bitmapfont will be null.

Reply:
I just realized that this means that the getFonts() will occur upon creating the first CameraStageBaseScreen, which means - upon creating the loading screen, and not "while loading", which could lead to many ANRs, as there were before the loading screen existed 0_0

Reply:
yes, you get it.😃

Reply:
This is Not Good =/
If that's what this means, then this is not a change that I want to occur